src/main/java/frc/robot/subsystems/funnel/commands/StartFunnel.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.subsystems.funnel.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.funnel.Funnel;
+
+/**
+ * this command activates Funnel's motor
+ */
+public class StartFunnel extends CommandBase {

```

Comment:
Add another command that does outtake

Reply:
He can do it in the command, as long as he add another parameter for the power, and bind it to 2 buttons